### DISCUSSION TITLE
Change card background color innstead of shape (circle)

### DISCUSSION THREAD

**Discussion by areyoufserious** [piitaya/lovelace-mushroom#957] [Ideas]
@piitaya coukd you please add setting to change card background color instead of shape?

and what i should change in code ot get it working?

**Comment by Lockie85** [comment]
I'd also like to do this, for a more minimal approach sometimes. For example, these have toggle controls for lights:
![image](https://user-images.githubusercontent.com/7091582/223195736-16f6d07e-af22-4b3b-bda5-6e495b05092c.png)
It would be great when on, if the background changed to orange and when off went back to white.
